pkg/pgmodel/model/pgutf8str/text_types.go

```diff
@@ -0,0 +1,147 @@
+// This file and its contents are licensed under the Apache License 2.0.
+// Please see the included NOTICE for copyright information and
+// LICENSE for a copy of the license.
+
+package pgutf8str
```

Comment:
optional: can we add a short documentation for the package describing which problem it solves.